### PR TITLE
upgrade Pomerium to v0.18

### PR DESCRIPTION
**Pull request by wasaga**
## Summary

Upgrades Pomerium to v0.18

## Related issues

**Checklist**:
- [ ] add related issues
- [ ] update Configuration in README
- [ ] update Changelog in README (major versions)
- [ ] update Upgrading in README (breaking changes)
- [x] ready for review

### CODE REVIEW

**Review by calebdoxsey — DISMISSED** [review]

File: charts/pomerium/README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@
   - [Redis Subchart](#redis-subchart)
   - [Configuration](#configuration)
   - [Changelog](#changelog)
+    - [32.0.0]()
```
Comment (desimone) [inline charts/pomerium/README.md:23]: Please describe changes

Comment (desimone) [reply charts/pomerium/README.md:23]: Update link?

File: charts/pomerium/templates/ingress-controller-deployment.yaml [hunk 2]
```diff
@@ -62,9 +63,6 @@ spec:
           - --databroker-tls-ca-file=/pomerium/ca/ca.crt
           - --metrics-bind-address=:8080
           - --health-probe-bind-address=:8081
-          {{- if or .Values.config.insecure .Values.config.insecureProxy }}
```
Comment (desimone) [inline charts/pomerium/templates/ingress-controller-deployment.yaml:65]: Can you describe this change and add that description to the changelog?

**Review by desimone — CHANGES_REQUESTED** [review]
Many of the markdown changes are whitespace / editor formatting. Can you either back out of those changes, or drop a config for whatever prettier/editor `rc` file so that we don't have future editor battles. Makes reviewing challening.

**Comment by wasaga** [comment]
Markdown is formatted with Prettier, similar to other repos. 
Repeating spaces were stripped automatically by it, there is no option to control that behaviour.

**Review by desimone — APPROVED** [review]

**Pull request merged**